src/module.ts

```diff
@@ -429,6 +543,50 @@ export default defineNuxtModule<ModuleOptions>({
       { name: 'useUnwrap', as: 'useUnwrap', from: resolveRuntimeModule('./composables/useUnwrap') }
     ])
 
+    if (options.search) {
+      const defaultSearchOptions: Partial<ModuleOptions['search']> = {
+        indexedSearch: true,
+        ignoredTags: ['style', 'code'],
+        ignoreDrafts: true,
+        ignoreEmpty: true,
+        ignorePartials: true,
+        // Maybe, we could rename it "indexedSearchOptions" since this will only be used for indexed search
+        options: {
+          fields: ['title', 'content', 'titles'],
+          storeFields: ['title', 'content', 'titles'],
+          searchOptions: {
+            prefix: true,
+            fuzzy: 0.2,
+            boost: {
+              title: 4,
+              content: 2,
+              titles: 1
+            }
+          }
+        }
+      }
+
+      options.search = {
+        ...defaultSearchOptions,
+        ...options.search
+      }
```

Comment:
Using `defu`  users will be able to just change a single field in a nested object without need to provide all fields
```suggestion
      options.search = defu(options.search, defaultSearchOptions)
```

Reply:
Yes but no. Doing this prevent user to overwrite a field like `fields` which is annoying.

Reply:
WDYT about replacing arrays?
https://github.com/nuxtlabs/studio-module/blob/869f61e2532141c258038fcbafb6c2271636afb9/src/runtime/composables/useStudio.ts#L14-L19